cmd/frontend/internal/httpapi/lsif.go

```diff
@@ -1,26 +1,157 @@
 package httpapi
 
 import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"strings"
+
+	"encoding/hex"
+	"encoding/json"
+
+	"fmt"
+
 	"net/http"
 	"net/http/httputil"
+	"net/url"
 
 	"github.com/gorilla/mux"
 	"github.com/sourcegraph/sourcegraph/cmd/frontend/backend"
+	"github.com/sourcegraph/sourcegraph/pkg/actor"
+	"github.com/sourcegraph/sourcegraph/pkg/api"
+	"github.com/sourcegraph/sourcegraph/pkg/env"
+	"github.com/sourcegraph/sourcegraph/pkg/extsvc/github"
+
+	"strconv"
 )
 
+var verificationGitHubToken = env.Get("LSIF_VERIFICATION_GITHUB_TOKEN", "", "The GitHub token that is used to verify that a user owns a repository.")
+var lsifUploadSecret = env.Get("LSIF_UPLOAD_SECRET", "", "Used to generate LSIF upload tokens. Must be long (160+ bits) to make offline brute-force attacks difficult.")
```

Comment:
```suggestion
var lsifUploadSecret = env.Get("LSIF_UPLOAD_SECRET", "", "Used to generate LSIF upload tokens. Must be long (20+ bytes) to make offline brute-force attacks difficult.")
```

Nobody measures characters in bits :)

How did you come to the conclusion 160 bits is sufficient?

Reply:
Switched to bytes 😸 It would cost a lot of money to crack a 160 bit password hashed with SHA-256 (looking at https://security.stackexchange.com/questions/168479/expert-quote-on-entropy-for-uncrackable-password)